File: sandbox.client/front/ts/model/ClientRecordListRequest.ts
import {ColumnSortType} from "./ColumnSortType";

export class ClientRecordListRequest {
  public clientRecordCountToSkip: number/*long*/;
  public clientRecordCount: number/*long*/;
  public columnSortType: ColumnSortType;
  public sortAscend: boolean;
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientRecordListRequest.java
package kz.greetgo.sandbox.controller.model;

import kz.greetgo.sandbox.controller.model.ColumnSortType;

public class ClientRecordListRequest {
  public long clientRecordCountToSkip;
  public long clientRecordCount;
  public ColumnSortType columnSortType;
  public boolean sortAscend;
}
